### ISSUE TITLE
Ability to create folders when exporting

### ISSUE THREAD

**Issue by shadownetdev1** [hydrusnetwork/hydrus#293]
It would be great if we could use / or \ to create folders in the export file name field.

This would allow us to do things like export all of the person namespaces to separate folders based on the value of the namespace.

Also it would be nice to be able to set an placeholder for when namespaces are blank but the export file name specifies the namespace. As of currently it just leaves this space empty which is sometimes undesirable.

**Comment by Zweibach** [comment]
Apparently creating folders on export isn't just a thing but has been for almost two years. The only thing really missing is a placeholder when the namespace is blank or missing.

Rename the title and edit the opening comment to reflect it or close this issue and open a new for placeholder namespaces in export.

**Comment by shadownetdev1** [comment]
If it is a thing then why does this:
![image](https://user-images.githubusercontent.com/26750646/87213306-f7e35880-c2f1-11ea-91a4-3349443325b6.png)

not make any folders and replace each / with a _ in the file name

**Comment by shadownetdev1** [comment]
It does the same for \

**Comment by shadownetdev1** [comment]
Opened a new issue for the placeholders as per your recommendation
#372 for reference

**Comment by Zweibach** [comment]
I thought you were talking about the export dialogue when right-clicking and going `share -> export -> file`. Either way, I went and tried it with export folders too and it worked the same way there. On Windows you need to use backslash `\`, Linux uses `/`.

Hydrus PTR and most parsers also parse artists into the `creator:` namespace. Not an issue if you use `artist:` locally or something but worth keeping in mind.

**Comment by shadownetdev1** [comment]
This comment made me realize that I had moved all my `artist:` namespaces to `creator:` namespaces. So I changed it to `creator:` and lo and behold it works.
I'm sorry for unnecessary taking up some of your time and am thankful to have figured it out thanks to your comment.
I still think that placeholders for empty namespaces should be a thing (they would have solved/hinted at my problem in this post) but since this thread is solved I will now close it.
Thank you again for your help @Zweibach
